Documentation/network/servicemesh/echo-app.rst

```diff
@@ -0,0 +1,21 @@
+Deploy the Echo App
+===================
+
+We will use a deployment made of echo servers.
+
+The application will reply to the client and, in the body of the reply, will include information about the pod and node receiving the original request. 
+We will use this information to illustrate how the traffic is manipulated by the Gateway. 
```

Comment:
This document needs to follow a similar format to other [service mesh entries](https://docs.cilium.io/en/stable/network/servicemesh/): prerequisites, installation (or deployment), and examples. So far, this looks like only the example portion of a new entry.

Reply:
I followed the same format as the [demo-app.rst](https://github.com/cilium/cilium/blob/main/Documentation/network/servicemesh/demo-app.rst) for this - it's not strictly a guide ; rather it's just a small embedded example app to be deployed as part of the guides.